### PR TITLE
Add CCP-NC Database

### PR DESCRIPTION
**Pull request by ml-evs**
This PR adds the upcoming OPTIMADE API for the [CCP-NC Database](https://www.ccpnc.ac.uk/database/) being implemented by @jkshenton and myself.

I also had to add a sleep to the CI so that validation only runs after netlify has sorted itself out.

### CODE REVIEW

**Review by merkys — DISMISSED** [review]
Looks good to me. I trust the GitHub checks to ensure validity of the added files, links themselves seem fine.

File: src/index-metadbs/ccpnc/v1/links.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+       "api_version" : "1.0.0",
+       "query" : {
+           "representation" : "/links"
+       },
+       "more_data_available" : false,
+       "schema" : "https://schemas.optimade.org/openapi/v1.0/optimade_index.json"
+   },
+   "data" : [
+      {
+         "id" : "cod",
```
Comment (ml-evs) [inline src/index-metadbs/ccpnc/v1/links.json:12]:
```suggestion
         "id" : "ccpnc",
```

**Comment by rartino** [comment]
Has this PR been sitting here since April? Sorry about that. Merge your 'cod' -> 'ccpnc' edit and I will approve.

**Comment by ml-evs** [comment]
> Has this PR been sitting here since April? Sorry about that. Merge your 'cod' -> 'ccpnc' edit and I will approve.

It has, but only because I've also been unable to do the underlying database work yet, so its not a problem!

**Comment by ml-evs** [comment]
Just had a meeting about this and its going to be a while until the base URL listed here goes live, so I will set it to `null` for now.

File: src/index-metadbs/ccpnc/v1/links.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+       "schema" : "https://schemas.optimade.org/openapi/v1.0/optimade_index.json"
+   },
+   "data" : [
+      {
+         "id" : "ccpnc",
+         "type" : "links",
+         "attributes" : {
+            "name": "CCP-NC Database",
+            "description": "The Collaborative Computational Project for NMR Crystallography (CCP-NC) Database hosts calculated NMR parameters submitted by users in the magres format.",
+            "base_url": "https://www.ccpnc.ac.uk/database/optimade/",
```
Comment (ml-evs) [inline src/index-metadbs/ccpnc/v1/links.json:17]:
```suggestion
            "base_url": null,
```

File: src/index-metadbs/ccpnc/v1/links.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+   },
+   "data" : [
+      {
+         "id" : "ccpnc",
+         "type" : "links",
+         "attributes" : {
+            "name": "CCP-NC Database",
+            "description": "The Collaborative Computational Project for NMR Crystallography (CCP-NC) Database hosts calculated NMR parameters submitted by users in the magres format.",
+            "base_url": null,
+            "homepage": "https://www.ccpnc.ac.uk/database/",
```
Comment (ml-evs) [inline src/index-metadbs/ccpnc/v1/links.json:18]:
```suggestion
            "aggregate": "staging",
```

File: src/index-metadbs/ccpnc/v1/links.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+   },
+   "data" : [
+      {
+         "id" : "ccpnc",
+         "type" : "links",
+         "attributes" : {
+            "name": "CCP-NC Database",
+            "description": "The Collaborative Computational Project for NMR Crystallography (CCP-NC) Database hosts calculated NMR parameters submitted by users in the magres format.",
+            "base_url": null,
+            "aggregate": "staging",
```
Comment (ml-evs) [inline src/index-metadbs/ccpnc/v1/links.json:18]:
```suggestion
            "aggregate": "staging",
            "homepage": "https://www.ccpnc.ac.uk/database",
```

**Review by rartino — APPROVED** [review]

**Pull request merged**